README.md

```diff
@@ -8,18 +8,30 @@ It provides SDF models of universal robot for Ignition Gazebo. In addition, it a
 
 ### Install
 
-* ROS: galactic
-* Ignition : Edifice 
+* ROS 2: Galactic
+* Ignition : Fortress 
 
 ```bash
 #install dependency
 sudo apt-get install ros-galactic-joint-state-publisher-gui
+export IGNITION_VERSION=fortress
 #cd src of ros2 workspace and download source code
 git clone https://github.com/gezp/universal_robot_ign.git
 #cd ros2 workspace and build
 colcon build
 ```
 
+Add the following entries to your .bashrc and replace YOUR_WORKSPACE with your workspace name.

```

Comment:
i think it's unnecessary, because it's basic concept of ROS which is from ros tutorial web.

Reply:
You're right, but for beginners that can be helpful.

Reply:
I've changed the readme accordingly.